src/libraries/System.Formats.Tar/src/System/Formats/Tar/TarEntry.cs

```diff
@@ -138,10 +138,10 @@ public string LinkName
         /// <remarks>The value in this field has no effect on Windows platforms.</remarks>
         public UnixFileMode Mode
         {
-            get => (UnixFileMode)_header._mode;
+            get => (UnixFileMode)(_header._mode & 0xFFF); // mask to only keep least significant 12-bits
```

Comment:
Can you update the comment so it reflects we're returning the `UnixFileMode` bits?

There's a `const` for these, but you probably can't access it here:

https://github.com/dotnet/runtime/blob/3f600a1afbb19034ec533b3b145ddb5b741623f0/src/libraries/System.Private.CoreLib/src/System/IO/FileSystem.cs#L8

Reply:
Sure, I will define this constant in S.F.Tar to keep the uniform style.